ts/receiver/closedGroups.ts

```diff
@@ -111,6 +111,16 @@ export async function handleClosedGroupControlMessage(
     return;
   }
   if (type === Type.NEW) {
+    if (
+      !getConversationController()
+        .get(envelope.senderIdentity)
+        .isApproved()
```

Comment:
maybe make sure that that the conversation exists first by checking for `?.isApproved()`

Reply:
also, you should drop that message from the cache with envelope. confirm() or removeFromCache(), one or the other I forgot